tcms/testruns/templates/testruns/get.html

```diff
@@ -63,39 +63,52 @@ <h2 class="card-pf-title" style="text-align: left">
                     </h2>
                 {% endif %}
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-spinner"></span>{% trans 'Status' %}:
-                    <input class="bootstrap-switch" id="status_button" type="checkbox"
-                        {% if not object.stop_date %}checked{% endif %}
-                        {% if not perms.testruns.change_testrun %}disabled{% endif %}
-                        data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
-                </h2>
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
                     {{ object.planned_start|default:'-'}}
                 </h2>
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
-                </h2>
+{#                Start date #}
+                {% if object.start_date == None %}
+                    <h2 class="card-pf-title" style="text-align: left">
+                        <span class="fa fa-spinner"></span>{% trans 'Start date' %}:
+                        <input class="bootstrap-switch" id="start_button" type="checkbox"
+
+                               {% if not perms.testruns.change_testrun %}disabled{% endif %}
+                               data-on-text="{% trans 'Started' %}" data-off-text="{% trans 'Planned' %}">
+                    </h2>
+                {% else %}
+                    <h2 class="card-pf-title" style="text-align: left">
+                        <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
+                        <span class="start-date">
+                            {{ object.start_date|default:'-'}}
+                    </span>
```

Comment:
Don't repeat the file definition! Only surround the button with conditional template tags.